Update learn-treasury.md (#744)
w3f/polkadot-wiki: docs/learn-treasury.md

In order for Bob to be paid his tip, some account must call the `close_tip` extrinsic at the end of
the closing period for the tip. This extrinsic may be called by anyone.

## Bounties Spending

> There are practical limits to Council Members curation capabilities when it comes to treasury proposals: Council members likely do not have the expertise to make a proper assessment of the activities described in all proposals. Even if individual Councillors have that expertise, it is highly unlikely that a majority of members are capable in such diverse topics.

Reviewer: ```suggestion
There are practical limits to Council Members curation capabilities when it comes to Treasury proposals: Council members likely do not have the expertise to make a proper assessment of the activities described in all proposals. Even if individual Councillors have that expertise, it is highly unlikely that a majority of members are capable in such diverse topics.
```

> Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem. 

Reviewer 2: ```suggestion
Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy, or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem. 
```

> Bounties Spending proposals aim to delegate the curation activity of spending proposals to experts called Curators: They can be defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy, or monitoring a set of tasks related to a specific topic: all for the benefit of the Polkadot ecosystem. 

Reviewer: ```suggestion
Bounties spending proposals aim to delegate the curation activity of spending proposals to experts called Curators. Curators are defined as addresses with agency over a portion of the treasury with the goal of fixing a bug, vulnerability, develop a strategy, or monitoring a set of tasks related to a specific topic for the benefit of the Polkadot ecosystem. 
```

> A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined after whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront payment to take the position. This deposit can be used to punish them if they act maliciously. But if they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back and part of the bounty reward.

Reviewer 2: ```suggestion
A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined later, whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront payment to take the position. This deposit can be used to punish them if they act maliciously. But if they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back and part of the bounty reward.
```

> A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined later, whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront payment to take the position. This deposit can be used to punish them if they act maliciously. But if they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back and part of the bounty reward.

Reviewer: ```suggestion
A proposer can submit a bounty proposal for the Council to pass, with a curator to be defined later, whose background and expertise is such that they are capable of determining when the task is complete. Curators are selected by the Council after the bounty proposal passes, and need to add an upfront deposit to take the position. This deposit can be revoked if the curator is found to have acted maliciously.  If they are successful in their task of getting someone to complete the bounty work, they will receive their deposit back as well as the bounty reward.
```

> When submitting the value of the bounty, the proposer includes a reward for curators willing to invest their time and expertise in the task: this amount is included in the total value of the bounty. In this sense, the curator's fee can be defined as the result of subtracting the value paid to the bounty rewardee from the total value of the bounty.

Reviewer: ```suggestion
When submitting the value of the bounty, the proposer includes a reward for curators willing to invest their time and expertise in the task. This amount is included in the total value of the bounty. The curator's fee can thus be defined as the result of subtracting the value paid to the bounty rewardee from the total value of the bounty.
```

> In general terms, curators are expected to have a well-balanced track record related to the issues the bounty tries to resolve: they should be at least knowledgeable on the topics the bounty touches, and show project management skills or experience. These recommendations ensure an effective use of the mechanism. A Bounty Spending is a reward for a specified body of work - or specified set of objectives - that needs to be executed for a predefined treasury amount to be paid out. The responsibility of assigning a payout address once the specified set of objectives is completed is delegated to the curator.

Reviewer: ```suggestion
In general terms, curators are expected to have a well-balanced track record related to the issue the bounty tries to resolve. They should be knowledgeable on the topics the bounty touches, and ideally also show project management skills or experience. A Bounty Spending is a reward for a specified body of work - or specified set of objectives - that needs to be executed for a predefined Treasury amount to be paid out. The responsibility of assigning a payout address once the specified set of objectives is completed is delegated to the curator.
```

> After the Council has activated a bounty, it delegates the work that requires expertise to the curator who gets to close the active bounty. Closing the active bounty enacts a delayed payout to the payout address and a payout of the curator fee. The delay phase allows the Council to act if any issues arise.

Reviewer: ```suggestion
After the Council has activated a bounty, it delegates the work that requires expertise to the curator who gets to close the active bounty. Closing the active bounty enacts a delayed payout to the payout address and a payout of the curator fee. The delay phase allows the Council to act if any issues arise immediately after payment.
```

> To minimize storage on chain in the same way as any proposal, bounties don't contain contextual information. When a user submits a bounty spending proposal, they will probably need to find an off-chain way to explain the proposal (any of the available community forums serve this purpose). [This template](https://docs.google.com/document/d/1-IBz_owspV5OcvezWXpksWDQReWowschD0TFuaVKKcU/edit?usp=sharing) can help as a checklist of all needed information for the Council to make an informed decision.

Reviewer: ```suggestion
To minimize storage on chain in the same way as any proposal, bounties don't contain contextual information. When a user submits a bounty spending proposal, they will probably need to find an off-chain way to explain the proposal (any of the [available community forums](community) serve this purpose). [This template](https://docs.google.com/document/d/1-IBz_owspV5OcvezWXpksWDQReWowschD0TFuaVKKcU/edit?usp=sharing) can help as a checklist of all needed information for the Council to make an informed decision.
```

Reviewer: Ideally we'd move the template to IPFS or some other more permanent way to access it but it's not necessary.

Author: do we need to add a link here for the community forums?

> The bounty has a predetermined duration of 90 days with the possibility of being extended by the curator. Aiming to maintain flexibility on the tasks’ curation, the curator will be able to create sub-bounties for more granularity and allocation in the next iteration of the mechanism.    

Reviewer: ```suggestion
The bounty has a default duration of 90 days with the possibility of being extended by the curator. In the future, the curator will be able to create sub-bounties for more granularity and fine-grained allocation.
```

### Creating a Bounty Proposal

> Anyone can create a Bounty proposal using Polkadot JS Apps: Users will be able to submit a proposal on the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way and it will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now, the help of a Councillor is needed to open a bounty proposal as a motion to be voted. 

Reviewer: ```suggestion
Soon, anyone will be able to create a Bounty proposal using Polkadot-JS. Users will be able to submit a proposal on the dedicated [Bounty](https://polkadot.js.org/apps/#/bounties) page. Currently, this page can only be used to view active bounties. The development of a more interactive user interface to both view and manage bounties in Polkadot-JS is currently being worked on, itself as a bounty. For now, the help of a Councillor is needed to open a bounty proposal as a motion on which to be voted. 
```

> Anyone can create a Bounty proposal using Polkadot JS Apps: From the website, use the Extrinsics tab and select the Treasury pallet, then `Propose_bounty` and enter the value and description of the proposal (usually linked to a forum). In the near future, users can use the dedicated Bounty section under Governance. The development of a robust user interface to view and manage bounties in the Polkadot Apps is on its way: It will serve Council members, Curators and Beneficiaries of the bounties, as well as all Kusama users observing the on-chain treasury governance. For now, 

Reviewer 2: Unfinished sentence at the end?

> A bounty can be cancelled by deleting the earmark for a specific treasury amount or be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active. 

Reviewer: ```suggestion
A bounty can be cancelled by deleting the earmark for a specific treasury amount, or closed if the task has been completed. The period for a bounty to be completed can also be extended by amending the expiry block number of the bounty to a higher block number. 
```

> A bounty can be cancelled by deleting the earmark for a specific treasury amount of be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active. 

Reviewer 2: ```suggestion
A bounty can be cancelled by deleting the earmark for a specific treasury amount or be closed if the tasks have been completed. On the opposite side, the 90 days life of a bounty can be extended by amending the expiry block number of the bounty to stay active. 
```

### Closing a bounty

> The curator can close the bounty once they approve the completion of its tasks. The curator should make sure to set up the payout address on the active bounty beforehand. Closing the Active bounty enacts a delayed payout to the payout address and a payout of the curator fee.

Reviewer: ```suggestion
The curator can close the bounty once they approve the completion of all of the associated tasks. The curator should be sure to set up the payout address on the active bounty beforehand. Closing an active bounty enacts a delayed payout to the payout address as well as the payout of the curator fee.
```

> The curator can close the bounty once it approves the completion of its tasks. The curator should make sure to set up the payout address on the active bounty beforehand. Closing the Active bounty enacts a delayed payout to the payout address and a payout of the curator fee.

Reviewer 2: ```suggestion
The curator can close the bounty once they approve the completion of its tasks. The curator should make sure to set up the payout address on the active bounty beforehand. Closing the Active bounty enacts a delayed payout to the payout address and a payout of the curator fee.
```

> A bounty can be closed by using the extrinsics tab and selecting the Treasury pallet, then `Award_bounty`, making sure the right bounty is to be closed and finally sign the transaction. It is important to note that those who received a reward after the bounty is completed, must claim the specific amount of the payout from the payout address, by calling `Claim_bounty` after the curator closed the allocation. 

Reviewer: ```suggestion
A bounty can be closed by using the [extrinsics](https://polkadot.js.org/apps/#/extrinsics) page and selecting `treasury.awardBounty`, setting the correct bounty ID and beneficiary, and then submitting and signing the extrinsic. The account that received a reward after the bounty is completed must claim the payout themself, by calling `treasury.claimBounty` after the curator has awarded the bounty. 
```

> A bounty can be closed by using the extrinsics tab and selecting the Treasury pallet, then `Award_bounty`, making sure the right bounty is to be closed and finally sign the transaction. It is important to note that those who received a reward after the bounty is completed, must claim the specific amount of the payout from the payour address, by calling `Claim_bounty` after the curator closed the allocation. 

Reviewer 2: ```suggestion
A bounty can be closed by using the extrinsics tab and selecting the Treasury pallet, then `Award_bounty`, making sure the right bounty is to be closed and finally sign the transaction. It is important to note that those who received a reward after the bounty is completed, must claim the specific amount of the payout from the payout address, by calling `Claim_bounty` after the curator closed the allocation. 
```

> To understand more about Bounties and how this new mechanism works, visit [Polkadot Blog](https://polkadot.network/kusama-and-polkadot-now-reward-curators-helping-to-scale-councils-functions-join-the-force-moving-the-community-forward/). 

Reviewer 2: ```suggestion
To understand more about Bounties and how this new mechanism works, read this [Polkadot Blog post](https://polkadot.network/kusama-and-polkadot-now-reward-curators-helping-to-scale-councils-functions-join-the-force-moving-the-community-forward/). 
```
